package.json

```diff
@@ -52,6 +52,7 @@
     "babel-core": "^5.6.15",
     "babel-loader": "^5.3.1",
     "babel-runtime": "^5.8.25",
+    "bootstrap": "^3.3.5",
```

Comment:
Do you all shrinkwrap your dependencies or lock down versions to prevent accidental auto-update issues?